package.json

```diff
@@ -196,7 +197,7 @@
       "applicationId": "bitwardendesktop",
       "identityName": "8bitSolutionsLLC.bitwardendesktop",
       "publisher": "CN=14D52771-DE3C-4886-B8BF-825BA7690418",
-      "publisherDisplayName": "8bit Solutions LLC",
+      "publisherDisplayName": "Bitwarden Inc",
```

Comment:
our publisher display name with windows store is still 8bit solutions llc

Reply:
Ah, I hadn't removed any of my testing code yet. Thanks for that catch!